librz/core/cmd_type.c

```diff
@@ -1933,6 +2047,24 @@ RZ_IPI RzCmdStatus rz_type_noreturn_del_all_handler(RzCore *core, int argc, cons
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_type_load_file_handler(RzCore *core, int argc, const char **argv) {
+	const char *path = argc > 1 ? argv[1] : NULL;
+	types_load_file(core, path);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_type_load_editor_handler(RzCore *core, int argc, const char **argv) {
+	const char *typename = argc > 1 ? argv[1] : NULL;
+	types_load_editor(core, typename);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_type_load_sdb_handler(RzCore *core, int argc, const char **argv) {
+	const char *path = argc > 1 ? argv[1] : NULL;
+	types_load_sdb(core, path);
```

Comment:
just do `types_load_sdb(core, argv[1])` and remove the previous line. This argument is anyway required.